### ISSUE TITLE
Supermarket /status endpoint throws error with PG 9.6+

### ISSUE THREAD

**Issue by teknofire** [chef/supermarket#1829]
# Describe the problem
The /status endpoint for Supermarket throws an error when connected to Postgresql 9.6+ 
```
{"message":"PG::UndefinedColumn: ERROR: column \"waiting\" does not exist\nLINE 1: select count(*) from pg_stat_activity where waiting='t'\n ^\n: select count(*) from pg_stat_activity where waiting='t'"}
```

It looks like the issue is with: https://github.com/chef/supermarket/blob/8403f4fc573c00403cc2c0e8f7a49494aa1750df/src/supermarket/app/lib/supermarket/health.rb#L71

The schema for `pg_stat_activity` changed with 9.6 and no longer has a waiting column.  Instead the same query would be 

```
SELECT count(*) from pg_stat_activity WHERE wait_event is not NULL;
```

## Software Version
Postgresql: 9.6+
Supermarket: 3.3.7

## Replication Case
1. Install Supermarket and use an external postgresql data with version >= 9.6
2. `curl https://FQDN/status`
3. Should get an error

## Stacktrace

<details><summary>  </summary>
```
```
</details>

**Comment by robbkidd** [comment]
Research:

```
wait_query = if ActiveRecord::Base.connection.postgresql_version < 90600
                "select count(*) from pg_stat_activity where waiting='t'"
              else
                "select count(*) from pg_stat_activity WHERE wait_event is not NULL"
              end
```

Horrah! No need for another round-trip query to detect PG server version! Based on [Rails' own PG server version detection](
https://github.com/rails/rails/blob/v5.2.4.4/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L320-L335).

**Comment by tas50** [comment]
This fix shipped in 3.4.8 today.
